governance/tag-governance.md

```diff
@@ -0,0 +1,541 @@
+- Ensure high-velocity completion of scoped effort with clear outcomes aligned with the CNCF mission
+  and TOC vision
+
+### Requirements
+
+- MUST be documented, including goals, milestones, deliverables, and any exit criteria (e.g., a
+  tracking GitHub issue).  
+- Initiatives MUST have more than two participants not including the assigned lead for oversight.  
+- SHOULD actively be worked on. An initiative that has made little progress in 3 months should be
+  canceled or paused until other contributors can be identified.   
+- Initiatives SHOULD not last longer than 2 quarters but MAY request an extension from their parent
+  governing body with reasonable justification.  
+- MUST check in at least monthly to assess the initiative's continued need, progress, and ability
+  to meet.  
+- SHOULD acknowledge everyone who actively helped complete the initiative.  
+- Subproject based initiatives MAY have different requirements that are outlined in the subproject
+  charter.
+
+### Roles
+
+ Assigned Tech Leads, Subproject Leads or TOC members serve as the general oversight of initiatives
+ depending on the parent governing body.
+
+### Formation
+
+There are three types of initiatives with different formation requirements, TAG initiatives,
+subproject initiatives, and TOC initiatives.
+
+#### TAG Initiatives
+
+General initiatives can be any "piece of work," such as white papers, reports, etc., that align
+with the scope of a TAG. The Tech Leads manage TAG Initiatives and MUST have at least one assigned
+as the sponsor to oversee the initiative's execution. They MAY delegate some of the responsibility
+of organizing and shepherding the initiative, but are still ultimately responsible for it.
+
+TAG initiatives can span multiple TAGs when something spans multiple tech domains, however one is
+ultimately assigned as the responsible group for execution and reporting. Initiatives that span
+many TAGs, or have no clear owner should be proposed as TOC initiatives. 
+
+#### Subproject Initiatives
+
+Subproject-based initiatives are defined in the subproject’s charter and are tailored to meet the
+subproject’s needs. They are essentially a customized GitHub issue template used to track the work
+of the subproject.
+
+#### TOC Initiatives
+
+The TOC may create and own initiatives when the scope of work spans the majority of TAGs or
+coordinate work around specific goals. In these instances, at least one TOC member MUST be assigned
+to organize, shepherd, and report on the initiative's status.
+
+## Technical Community Groups
+
+Technical Community Groups are a part of the broader [CNCF Community Groups Program]. They are
+topic and domain-focused or locality-based groups used as a rallying point for community members
+to discuss, share knowledge, and coordinate potential future initiatives. They are free to propose
+and shepherd initiatives when there is an explicit deliverable or objective they wish to work on.
```

Comment:
Who owns CG initiatives?

Reply:
They would be proposed and reviewed by the most appropriate TAG or or TOC post triage.